frontend/src/index.scss

```diff
@@ -11,6 +11,7 @@
 @use "components/Carousel/ScrollCarousel";
 @use "components/Navigation/HeaderNav";
 @use "components/Navigation/InnerCopNav";
+@use "components/Navigation/Footer/_Footer.scss";
```

Comment:
I usually create a structure in the way that if a component has subcomponents then these subcomponents should be under a folder with component's name.

Reply:
Why?

Reply:
Would you like to think over your proposed nesting structure then until you are ready for a review?

To be honest, I arbitrarily decided upon the existing nesting structure, so it is not made with anyone else but me in mind. I decided to just mirror MUI, with 3 levels of nesting. If the team agrees there is a better way, I am more than fine with shouldering the cost of making the transition. But I am reluctant if there is not a strong reason.

For reference, this is the cost of changing our nesting structure:

1. The further nested structure is not import friendly as the url path tunnel into the Footer twice. It also breaks the existing architecture, which is not intuitive to work with.
2. In order to address the issue, we will need to open up a meeting to discuss the best way to mitigate these problems and come up with a solution we can agree upon.
3. After coming up with a solution, we need to open new issues to edit the code and our documentation. The issue will affect multiple files, which raises a high risk of merge conflicts.
4. While we wait for the issues to be resolved, we are in a limbo where we need to explain to future new developer that the wiki is outdated. The whole team as well would also need to deal with a messy import structure as they work since some are 3 levels, while others are 4 levels deep.

Anyways, please consider these concerns and give me feedback on the best way to proceed. One of my priorities as a lead is to create a developer friendly environment, so I am happy to discuss thoughts and opinions.